fastlane/fastlane
# [action] add action google_play_track_release_names

* Allows user to retrieve a list of release names for a track from google play

### Checklist

### Motivation and Context
Before performing a new APK upload it can be useful to check existing release names, or as part of a CI process to automate version naming. This action complements the existing google_play_track_version_codes action.

### Description
Added the google_play_track_release_names with similar implementation as the existing google_play_track_version_codes action

### Testing Steps
Run the action `google_play_track_release_names` and observe an array of release names for the given track
